tests/spec/core/idl-index-spec.js

```diff
@@ -40,6 +40,153 @@ interface Bar {
     expect(header.textContent).toEqual("1. IDL Index");
   });
 
+  //  this should exclude because it has an "exclude" class declared
+  //  or the IDL parent section is informative (non-normative)
+  it("excludes things the editor doesn't want in the idl summary and informative class", async () => {
+    const body = `
+      ${makeDefaultBody()}
+      <section>
+        <pre class="idl exclude">
+        interface Foo {
+          readonly attribute DOMString bar;
+        };
+        </pre>
+      </section>
+      <section>
+        <pre class=idl>
+        interface Bar {
+          readonly attribute DOMString foo;
+        };
+        </pre>
+      </section>
+      <section class="informative">
+        <pre class="idl">
+        interface Baz {
+          readonly attribute DOMString baz;
+        };
+        </pre>
+      </section>
+      <section id="idl-index"></section>
+    `;
+    const ops = {
+      config: makeBasicConfig(),
+      body,
+    };
+    const doc = await makeRSDoc(ops);
+    const idlIndex = doc.getElementById("idl-index");
+    expect(idlIndex.innerText.includes("Foo")).toEqual(false);
+    expect(idlIndex.innerText.includes("Baz")).toEqual(false);
```

Comment:
Would be great to also check `.toEqual(true)`.

Reply:
Agree, it's good to check for the false positives too... to assure that what should be there is actually there.